doc/sphinx/language/gallina-extensions.rst

```diff
@@ -591,7 +591,7 @@ The following command is available when the ``FunInd`` library has been loaded v
 .. cmd:: Function @fix_definition {* with @fix_definition }
 
    This command is a generalization of :cmd:`Fixpoint`. It is a wrapper
-   for several ways of defining a function *and other useful related
+   for several ways of defining a function and other useful *related
```

Comment:
I think the point was to insist on the *and* (it doesn't just define a function).

Reply:
I put it only on the "and".  What do you think?

BTW, the italic doesn't stand out very much.  Bold would definitely be noticeable.

![image](https://user-images.githubusercontent.com/1253341/76564718-5ee92180-6466-11ea-92ad-2ed9fa25db6a.png)